app/src/main/java/com/example/android/notekeeper/adp/NoteAdp.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.android.notekeeper.adp
+
+import android.content.Context
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.android.notekeeper.NoteInfo
+import com.example.android.notekeeper.R
+import kotlinx.android.synthetic.main.item_note.view.*
+
+class NoteAdp(private val noteInfoList: List<NoteInfo>, private val context: Context, private val listener: Listener) :
```

Comment:
You can remove this context and use it from the view.

Reply:
Sure. My mistake. Done